fixtures/minuit-une/ivl-photon.json

```diff
@@ -0,0 +1,301 @@
+      "defaultValue": 0,
+      "capability": {
+        "type": "WheelSlot",
+        "slotNumber": 1
+      }
+    },
+    "Gobo Rot": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 12],
+          "type": "WheelSlot",
+          "slotNumber": 1,
+          "comment": "OFF"
+        },
+        {
+          "dmxRange": [13, 122],
+          "type": "WheelSlot",
+          "slotNumber": 2,
+          "comment": "Spin CW"
+        },
+        {
+          "dmxRange": [123, 133],
+          "type": "WheelSlotRotation",
+          "speed": "stop",
+          "comment": "Rel"
+        },
+        {
+          "dmxRange": [134, 242],
+          "type": "WheelSlot",
+          "slotNumber": 4,
+          "comment": "Spin CCW"
+        },
+        {
+          "dmxRange": [243, 255],
+          "type": "WheelSlot",
+          "slotNumber": 5,
+          "comment": "OFF"
+        }
+      ]
+    },
+    "Gobo Size": {
+      "fineChannelAliases": ["Gobo Size Fine"],
+      "defaultValue": 0,
+      "capability": {
+        "type": "WheelSlot",
+        "slotNumber": 1
+      }
+    },
+    "Zoom": {
+      "fineChannelAliases": ["Zoom Fine"],
+      "defaultValue": 0,
+      "capability": {
+        "type": "Generic"
+      }
+    },
+    "Zoom spd": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Generic",
+        "comment": "Zoom Speed"
+      }
+    },
+    "TILT": {
+      "fineChannelAliases": ["Tilt Fine"],
+      "defaultValue": 0,
+      "capability": {
+        "type": "Tilt",
+        "angleStart": "0%",
+        "angleEnd": "100%",
+        "helpWanted": "Can you provide exact angles?"
+      }
+    },
+    "Dimmer": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Red": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red",
+        "comment": "Red C-Mix"
+      }
+    },
+    "Green": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green",
+        "comment": "Green C-Mix"
+      }
+    },
+    "Blue": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue",
+        "comment": "Blue C-Mix"
+      }
+    }
+  },
+  "modes": [
+    {
+      "name": "STANDARD",
+      "channels": [
+        "Control",
+        "Shutter",
+        "Shutter range",
+        "Gobo Split",
+        "Gobo Index",
+        "Index Fine",
+        "Gobo Rot",
+        "Gobo Size",
+        "Gobo Size Fine",
+        "Zoom",
+        "Zoom Fine",
+        "Zoom spd",
+        "TILT",
+        "Tilt Fine",
+        "Dimmer",
+        "Red",
+        "Green",
+        "Blue"
+      ]
+    },
+    {
+      "name": "TILT Cell 1",
+      "channels": [
+        "TILT",
+        "Tilt Fine",
+        "Dimmer",
+        "Red",
+        "Green",
+        "Blue"
+      ]
+    }
```

Comment:
This looks like a rough draft.  #3692 has the correct channel mapping.  I suggest we close this PR, and make any necessary fixes to #3692 (which already looks pretty good).

Reply:
Thanks @kengruven I'll close this one :)